### DISCUSSION TITLE
quality of grain orientation map

### DISCUSSION THREAD

**Discussion by koushik-wihg** [mtex-toolbox/mtex#1253] [Ask Anything] [answered]
Hi I am preparing grain orientation map for individual grain (with maximum grain orientation spread) for omphacite from eclogitic rocks.
using the following script

% find the grain with maximum grain orientation spread
[~,id] = max(grains.GOS);
grain_selected = grains(id)

% plot the grain with its orientations
close all
plot(grain_selected.boundary,'linewidth',2)
hold on
plot(ebsd(grain_selected),ebsd(grain_selected).orientations)
hold off

alternatively I have also used 
plot(ebsd(grain_selected),ebsd(grain_selected).orientations)

The problem I am facing is the quality of the image which is pixelated (although different color of pixels for different domains.

I have attached two such images. Need help for obtaining better quality images with different domains (subgrains) of the grain having different shades and also the whole grain covered.

Please check the attachments. 
![IMG_20211009_154053835_MP](https://user-images.githubusercontent.com/92208943/136654120-0f80991d-7b20-4fad-b93d-305386442bc1.jpg)
![IMG_20211009_154118429](https://user-images.githubusercontent.com/92208943/136654124-1d87dfc0-bce9-4314-ae49-4f034f55be0f.jpg)

Thanks in advance
Koushik

**Comment by zmichels** [comment, marked as answer]
Hi,

I’m not sure exactly what you mean by “quality” – can you be more specific in describing what it is you want your figure to look like? 
The main thing I notice is that it looks like there are many not-indexed pixels. If that is the issue, then the problem is likely with the quality of the data itself. You can try to fill the missing pixels – see the section on "Interpolating Missing Data” at this webpage: https://mtex-toolbox.github.io/HomepageOld/files/doc/EBSDSmoothing.html <https://mtex-toolbox.github.io/HomepageOld/files/doc/EBSDSmoothing.html>

Zach
